src/TTImp/ProcessRunElab.idr

```diff
@@ -25,6 +26,8 @@ processRunElab : {vars : _} ->
                  RawImp -> Core ()
 processRunElab eopts nest env fc tm
     = do defs <- get Ctxt
+         when (not (isExtension ElabReflection defs)) $
```

Comment:
Stylistic remark: `when (not b)` is `unless b`

Reply:
A quick count `grep "when (not" src/**/*.idr -c | awk -F':' '{ count += $2 } END { print count }'` returns 34 matches, do you think all should be changed to `unless` or just the new line?

Reply:
I don't think this PR has to change all of the occurrences but I think
it would make sense to avoid introducing new ones.

Reply:
Done. 👍